### PR TITLE
Updated pangram; added version file "1.4.1"

### PR DESCRIPTION
**Pull request by guygastineau**
[problem-specifications](https://github.com/exercism/problem-specifications/blob/master/exercises/pangram/canonical-data.json)

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: exercises/pangram/pangram_test.sh [hunk 1]
```diff
@@ -1,56 +1,63 @@
 #!/usr/bin/env bats
 
+# Check if the given string is an pangram
```
Comment (budmc29) [inline exercises/pangram/pangram_test.sh]: It's better to add a newline below this, to make it clean the comment if for the file and not the first test.

**Comment by guygastineau** [comment]
@budmc29

Do you want this PR to wait until your PR to problem-specs goes through?

I can just take pangram straight to version 1.4.1 then.

File: exercises/pangram/pangram_test.sh [hunk 2]
```diff
@@ -1,56 +1,63 @@
...
 @test "sentence empty" {
-    #skip
-    run bash pangram.sh ""
-    [ "$status" -eq 1 ]
+  #skip
+  run bash pangram.sh ""
+  [ "$status" -eq 1 ]
```
Comment (budmc29) [inline exercises/pangram/pangram_test.sh]: This test is missing an output validation, meaning that you can make the test pass (which is currently the case with the example file), without making the tool usable from the command line.

We had a discussion in the past that we should not rely on the exit status, but actually return something (like "false" for example).
I don't remember if it was on a PR I made or an issue, so I can't link to it.

This applies to all the tests.

Comment (budmc29) [reply exercises/pangram/pangram_test.sh]: Your call with the status, I have nothing against it being used as well as long as we're outputting something.

Comment (budmc29) [reply exercises/pangram/pangram_test.sh]: Your call with the status, I don't have anything against it being used, as long as we also output something.

Comment (budmc29) [reply exercises/pangram/pangram_test.sh]: Your call with the status, I don't have anything against it being used, as long as we output something.

Sorry about that, silly comment system.

Comment (guygastineau) [reply exercises/pangram/pangram_test.sh]: I'll be home from work in an hour, so I'll get these changes in place.

**Comment by budmc29** [comment]
Thank you.

**Pull request merged**